stable/fluent-bit/templates/daemonset.yaml

```diff
@@ -97,9 +97,14 @@ spec:
           mountPath: /fluent-bit/etc/parsers_custom.conf
           subPath: parsers.conf
 {{- end }}
+{{- if and .Values.backend.type "stackdriver" .Values.backend.stackdriver.google_service_credentials }}
+        - name: fluentbit-secret
+          mountPath: /secure/google_service_credentials.json
+          subPath: google_service_credentials.json
+{{- end }}
 {{- end }}
 {{- if .Values.backend.es.tls_ca }}
-        - name: es-tls-secret
+        - name: fluentbit-secret
```

Comment:
why do you change the name of this secret ? let's keep the names as before

Reply:
I don't see the need to create another Secrets resource just to store the google cloud credentials.

Hence I renamed the existing Secrets file so it now both stores the google cloud credentials and the SSL cert used by elasticsearch.

Reply:
Changed back. See comment https://github.com/helm/charts/pull/15549#discussion_r318852224